### ISSUE TITLE
compute projections only after mcorr, use `get_parent()` if `get_projection()` called on cnmf item

### ISSUE THREAD

**Issue by kushalkolar** [nel-lab/mesmerize-core#182]
It's a waste of time re-computing projections if they're already done after mcorr, so if we're using a CNMF item we could just get the projections from the parent mcorr item.

this will require finally merging #119 soon...
